src/OpenColorIO/CMakeLists.txt

```diff
@@ -178,8 +178,23 @@ set(SOURCES
 
 set(prefix ${CMAKE_INSTALL_PREFIX})
 set(exec_prefix "\${prefix}")
-set(libdir "\${exec_prefix}/${CMAKE_INSTALL_LIBDIR}")
-set(includedir "\${exec_prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
+
+# CMAKE_INSTALL_LIBDIR is not guaranteed to be relative.
+# Not using cmake_path function since it is only available from CMake ≥ 3.20.
+if(IS_ABSOLUTE "${CMAKE_INSTALL_LIBDIR}")
+    set(libdir "${CMAKE_INSTALL_LIBDIR}")
+else()
+    set(libdir "\${exec_prefix}/${CMAKE_INSTALL_LIBDIR}")
+endif()
+
+# CMAKE_INSTALL_INCLUDEDIR is not guaranteed to be relative.
+# Not using cmake_path function since it is only available from CMake ≥ 3.20.
+if(IS_ABSOLUTE "${CMAKE_INSTALL_INCLUDEDIR}")
+    set(includedir "${CMAKE_INSTALL_INCLUDEDIR}")
+else()
+    set(includedir "\${exec_prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
```

Comment:
Should be `${prefix}` for include directory.

Reply:
I didn't want to change it since I don't know the reason why it was changed to ${exec_prefix} in the past. It was changed to ${exec_prefix} in [PR #1120](https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1120/files)

Pinging @remia

Reply:
It should be restored back then.

Reply:
I don't remember a specific reason I made this change and the build still works fine for me when I update that. Though I'm not sure what it changes exactly given that we set on line 180 the two to be the same thing? Or did I misunderstood the logic here (I didn't write this code)?

Reply:
The GNU documentation linked above is pretty clear for `libdir` and `includedir` prescriptions, they are different, e.g.
```
libdir=${exec_prefix}/lib
includedir=${prefix}/include
```

Reply:
Thanks for the link @kmilos. I'll make the change.

Reply:
Thanks, I think the pkg-config subject should be sorted for a while now ;)